src/plots/cartesian/layout_attributes.js

```diff
@@ -272,17 +272,69 @@ module.exports = {
     },
     autorange: {
         valType: 'enumerated',
-        values: [true, false, 'reversed'],
+        values: [true, false, 'reversed', 'min reversed', 'max reversed', 'min', 'max'],
         dflt: true,
         editType: 'axrange',
         impliedEdits: {'range[0]': undefined, 'range[1]': undefined},
         description: [
             'Determines whether or not the range of this axis is',
             'computed in relation to the input data.',
             'See `rangemode` for more info.',
-            'If `range` is provided, then `autorange` is set to *false*.'
+            'If `range` is provided and it has a value for both the',
+            ' lower and upper bound, `autorange` is set to *false*.',
```

Comment:
```suggestion
            'lower and upper bound, `autorange` is set to *false*.',
```
Sorry @archmoj, I guess this space should go for the scheme to generate with only one.